MOM_input

```diff
@@ -400,7 +419,7 @@ OCEAN_SURFACE_STAGGER = A
                                 ! returned to the coupler.  Valid values include
                                 ! 'A', 'B', or 'C'."
 
-RESTART_CONTROL = 3
+RESTART_CONTROL = 1
```

Comment:
It was agreed to change `RESTART_CONTROL =1` in the MOM parameter discussion meeting,  [refer here](https://forum.access-hive.org.au/t/namelist-configuration-discussion-meeting/1917/9#:~:text=RESTART_CONTROL%20%2D%20try%20using%20default%20(1))

Reply:
Do you know what this does? I haven't looked in detail but it looks like it just turns off timestamping the restart files, which I think would do nothing for us since the NUOPC cap sets the restart filename.

Reply:
Yes it is for time stamping and in our configs NUOPC does that, so we agreed to restore it to a default value which is 1

Reply:
Can we delete it ? :) Or put at worst, put a comment in MOM_input that it's not used?

Reply:
Yeah, delete it

Reply:
Deleted in this commit acbb068